src/aspire/eng/Build.props

```diff
@@ -1,18 +1,16 @@
 <Project>
-  <ItemGroup Condition="'$(DotNetBuildFromSource)' != 'true' and '$(DotNetBuild)' != 'true'">
+
+  <ItemGroup Condition="'$(DotNetBuild)' != 'true'">
```

Comment:
Aspire within the VMR is on 8.0 arcade...I don't think we can change this?

Reply:
Aspire inside the VMR builds with live arcade :)